pyproject.toml

```diff
@@ -1,28 +1,20 @@
 [build-system]
-requires = ["setuptools>=61", "wheel", "setuptools_scm>=6.2"]
-build-backend = "setuptools.build_meta"
+requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning>=1.0.0,<2.0.0"]
+build-backend = "poetry_dynamic_versioning.backend"
 
-[project]
+[tool.poetry]
 name = "locust"
-license = { text = "MIT" }
 description = "Developer-friendly load testing framework"
-dynamic = ["version"]
-requires-python = ">=3.9"
-dependencies = [
-    "gevent >=22.10.2",
-    "flask >=2.0.0",
-    "Werkzeug >=2.0.0",
-    "requests >=2.26.0",
-    "msgpack >=1.0.0",
-    "pyzmq >=25.0.0",
-    "geventhttpclient >=2.3.1",
-    "ConfigArgParse >=1.5.5",
-    "tomli >=1.1.0; python_version<'3.11'",
-    "psutil >=5.9.1",
-    "Flask-Login >=0.6.3",
-    "Flask-Cors >=3.0.10",
-    "pywin32; platform_system=='Windows'",
-]
+version = "0.0.0"
+license = "MIT"
+readme = "README.md"
+authors = ["Jonatan Heyman", "Carl Byström", "Joakim Hamrén", "Hugo Heyman"]
```

Comment:
Just took this info from the main site 😓

Reply:
Make it 

`authors = ["Jonatan Heyman", "Lars Holmberg", "Andrew Baldwin"]`

That's the most accurate without listing every contributor :)